src/widget/wtrackmenu.cpp

```diff
@@ -267,7 +266,7 @@ void WTrackMenu::createActions() {
     }
 
     if (featureIsEnabled(Feature::Properties)) {
-        m_pTrackInfo.reset(new DlgTrackInfo(m_pConfig, nullptr, m_pTrackModel));
+        m_pTrackInfo.reset(new DlgTrackInfo(nullptr, m_pConfig, m_pTrackModel));
```

Comment:
Shouldn't `this` be the parent?

Reply:
Ideally it should be. But another comment with initialization of m_pTagFetcher clarifies that this makes the dialog partially inherit some styles, which makes it kind of a hybrid between our stylesheets and system native style.

Reply:
Hmm, we could have the skins style DlgTrackInfo and DlgTagFetcher. We recently extended skin styles to other context menus and popups in #2363. @ronso0 what do you think? Do you want to add styles for DlgTrackInfo and DlgTagFetcher? If not, I think it would be fine for @hacksdump to leave these with the system's style. If we stick with the system style for these dialogs, please copy that comment here for the DlgTrackInfo construction.

Reply:
Yes, IMO it would a huge win to also style the Properties window: even though I have a dark OS theme, this window is not styled by it (for me it's really bright and very generously sized). I also have some visual tweaks in mind that make it easier to read.
I'll open a PR soonish.
The menu itself already looks as the WLibrary track menu :+1:

Reply:
@ronso0 I added a card to the 2.3 project in the To Do column so we don't forget. It would be good to do this for the 2.3 release along with the rest of the skin styles for other menus and popups. No need to rush to do it before the beta release though.